File: WEB v0.3/data/start.js
const DEBUG = true;

function startGame() {
    fetch("/startGame").then(function (response) {
        return response.text();
    }
    ).then(function (text) {
        if (DEBUG) console.log("startGame: " + text);
        if (text === "ok") window.location.href = "/game_page.html";
    });
}

File: WEB v0.3/src/start.cpp
#include "webConnection.hpp"

#define DEBUG false

int MyWebServer::initStartGameRequest()
{
    startGameRequest();
    return 0;
}

void MyWebServer::startGameRequest()
{
    server->on("/startGame", HTTP_GET, [this](AsyncWebServerRequest * request)
        {
            this->started=true;
            if(DEBUG)
                Serial.println("StartGame request:"+request->url()+" response:"+" ok");
            
            //send to all connected player that the game has started
            gameStartedSSE->send("start",NULL,millis(),100);
            //sand to the serial port that the game has started
            toSendToSerial += ("/startGame?playerList=" + constructPlayerList()+"\n");
            request->send(200,"text/plain","ok");
        }
    );
}